stratum/hal/lib/tdi/dpdk/dpdk_hal_test.cc

```diff
@@ -192,28 +192,14 @@ TEST_F(DpdkHalTest, SanityCheckFailureWhenPersistentConfigDirFlagNotGiven) {
 }
 
 TEST_F(DpdkHalTest, ColdbootSetupSuccessForSavedConfigs) {
-  // Setup and save the test config(s).
+  // Set up and save the test config(s).
   ChassisConfig chassis_config;
   ForwardingPipelineConfigs forwarding_pipeline_configs;
   FillTestChassisConfigAndSave(&chassis_config);
   FillTestForwardingPipelineConfigsAndSave(&forwarding_pipeline_configs);
 
   EXPECT_CALL(*switch_mock_, PushChassisConfig(EqualsProto(chassis_config)))
       .WillOnce(Return(::util::OkStatus()));
-  EXPECT_CALL(
```

Comment:
Does this need to be added to the warmboot routine?

Reply:
I don't know. I did the minimum necessary to address the issue, which is due to differences in the way we handle pipeline configuration files.

My guess is, probably not. This test was adapted from the one for the Barefoot platform. We assume it did everything necessary until proven otherwise. The coverage analysis for the test might answer the question.

In any case, the purpose of this PR is to repair a failing unit test so it passes. Improving it is different problem.